packages/SystemsThinkingWorkbench-Core.package/STWNode.class/instance/addHandlesTo.box..st

```diff
@@ -0,0 +1,53 @@
+halos and balloon help
+addHandlesTo: myHalo box: aBox
+	"overwrite the addHandles of the standart Morph and places our own Handles.
+	
+	Since I am a Circle, I won't use the Rectangle and directly add my Handles to my Halo.
+	The Box is only sent, to overwrite the function.
+	
+	A Handle is a simple Morph, who gets added onto the Halo and triggers events"
```

Comment:
Please apply style guidelines regarding line breaks.

The question still stands: what is a fulcrum.